cider/tests/functionality/CiderAggTest.cpp

```diff
@@ -106,6 +106,53 @@ TEST_F(CiderAggTest, sumTest) {
   // TODO: SUM(decimal) with half null
 }
 
+TEST_F(CiderAggTest, aggArrowTest) {
+  // SUM(int)
+  prepareArrowBatch();
```

Comment:
`QueryArrowDataGenerator ` should can be used for data generation now. We need to switch to it, you can refer [PR](https://github.com/intel/BDTK/pull/97) first.

Reply:
Done.